examples/middleware/.env

```
@@ -1,4 +1,4 @@
-NEXT_PUBLIC_STACK_API_URL=# enter your stack endpoint here, e.g. http://localhost:8102
-NEXT_PUBLIC_STACK_PROJECT_ID=# enter your stack project id here
-NEXT_PUBLIC_STACK_PUBLISHABLE_CLIENT_KEY=# enter your stack publishable client key here
-STACK_SECRET_SERVER_KEY=# enter your stack secret server key here
+NEXT_PUBLIC_HEXCLAVE_API_URL=# enter your stack endpoint here, e.g. http://localhost:8102
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)